### PR TITLE
LRQA-69018

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase [hunk 1]
```diff
@@ -170,6 +170,89 @@ definition {
 			key_position2 = "3");
 	}
 
+	@description = "Verify that a Form can be created with Spanish as the default portal language"
+	@priority = "5"
+	test CanBeCreatedWithSpanishAsDefaultLanguage {
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase:175]: @leticiachiu, can you please verify with Diogo if this test can be added to the Acceptance Suite?

This is an important test which has already helped Forms Team prevent bugs. :wink:

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase [hunk 2]
```diff
@@ -1322,6 +1683,79 @@ definition {
 			fieldName = "Selección Única");
 	}
 
+	@description = "LPS-116866 - Verify the Submit Button Label can be translated to multiple languages"
+	@priority = "4"
+	test SubmitButtonCanBeTranslated {
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase:1688]: @leticiachiu, how about rename to `SubmitButtonLabelCanBeTranslated`?

I believe it'll a little bit more consistent with the existing UI.

image

**Comment by leticiachiu** [comment]
@natocesarrego fixed :)

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
Hey @leticiachiu, can you please fix the source formatter issues and resend? Thanks

**Pull request closed without merging**